### ISSUE TITLE
Asciidoctorj gives error if input file is an absolute path

### ISSUE THREAD

**Issue by enr** [asciidoctor/asciidoctorj#275]
Hi,

I have error if I pass the input file as an absolute path, but using relative path everything is ok.

Asciidoctorj version (installed via GVM):

	enrico@lap:~/union/feasibility-docs/developers-manual$ asciidoctorj --version
	Asciidoctor 1.5.2 [http://asciidoctor.org]
	
The failing command:

	enrico@lap:~/union/feasibility-docs/developers-manual$ asciidoctorj --backend html5 \
	--destination-dir /home/enrico/union/feasibility-docs/developers-manual/build \
	--out-file index.html --doctype book --attribute project-version=0.1 \
	/home/enrico/union/feasibility-docs/developers-manual/src/asciidoc/index.adoc
	asciidoctor: FAILED: input file(s) '[/home/enrico/union/feasibility-docs/developers-manual/src/asciidoc/index.adoc]' missing or cannot be read
	Exception in thread "main" java.lang.IllegalArgumentException: asciidoctor: FAILED: input file(s) '[/home/enrico/union/feasibility-docs/developers-manual/src/asciidoc/index.adoc]' missing or cannot be read
	at org.asciidoctor.cli.AsciidoctorInvoker.invoke(AsciidoctorInvoker.java:51)
	at org.asciidoctor.cli.AsciidoctorInvoker.main(AsciidoctorInvoker.java:184)

The input file exists:

	enrico@lap:~/union/feasibility-docs/developers-manual$ ll /home/enrico/union/feasibility-docs/developers-manual/src/asciidoc/index.adoc
	-rw-rw-r-- 1 enrico enrico 288 gen 27 16:39 /home/enrico/union/feasibility-docs/developers-manual/src/asciidoc/index.adoc

The successful command (change only the input file path, from absolute to relative):

	enrico@lap:~/union/feasibility-docs/developers-manual$ asciidoctorj --backend html5 \
	--destination-dir /home/enrico/union/feasibility-docs/developers-manual/build \
	--out-file index.html --doctype book --attribute project-version=0.1 \
	src/asciidoc/index.adoc

Is this expected?

Anyway, thanks a lot for this awesome tool!

Enrico

**Comment by lordofthejars** [comment]
At least this is not the expected behaviour, we are going to check why this happens and fix it. 

Thank you very much for using asciidoctor.

**Comment by enr** [comment]
Ok, let me know if you need help to reproduce the problem or test the fix :)
Cheers

**Comment by mojavelinux** [comment]
I get the same result. Seems to happen even for a simple invocation with an absolute path:

    $ asciidoctorj /tmp/article.adoc

Results in:

```
asciidoctor: FAILED: input file(s) '[/tmp/article.adoc]' missing or cannot be read
Exception in thread "main" java.lang.IllegalArgumentException: asciidoctor: FAILED: input file(s) '[/tmp/article.adoc]' missing or cannot be read
	at org.asciidoctor.cli.AsciidoctorInvoker.invoke(AsciidoctorInvoker.java:51)
	at org.asciidoctor.cli.AsciidoctorInvoker.main(AsciidoctorInvoker.java:184)
```

**Comment by lordofthejars** [comment]
Fixed in master, now we need to rebase to asciidoctorj1.6.0 branch.

**Comment by mojavelinux** [comment]
:+1:

**Comment by robertpanzer** [comment]
Also fixed on asciidoctorj-1.6.0 with https://github.com/asciidoctor/asciidoctorj/commit/0f9cb5b2c437bb30126e40df1e09118b6375237d
